lib/gooddata/models/metadata.rb

```diff
@@ -282,5 +312,20 @@ def unlisted=(flag)
     def validate
       true
     end
+
+    def get_flag(flag)
+      meta['flags'] && meta['flags'].include?(flag)
+    end
+
+    def set_flag(flag, value)
+      meta['flags'] = [] unless meta['flags']
+
+      if (value == '1' || value == 1 || value == true) && !meta['flags'].include?(flag)
+        meta['flags'].push(flag)
+        meta['flags'].sort!
+      elsif !value && meta['flags'].include?(flag)
```

Comment:
could we replace `meta['flags'].include?(flag)` with `has_flag(flag)` for readability?